lace/emulator/nn_emulator.py

```diff
@@ -406,6 +406,15 @@ def _get_training_cov(self):
             training_cov = torch.Tensor(training_cov)
         return training_cov
 
+
+    def _get_rescalings_weights(self):
+        weights_rescalings = np.ones(len(self.training_data))
+        if self.emulator_label == "Nyx_alphap_cov": 
+            weights_rescalings[np.where([(d['val_scaling'] not in [0,1] and d['z'] in [2.8, 3,3.2,3.4]) for d in self.training_data])] = 1
```

Comment:
We should use ind_rescaling, not val_scaling. Remember I told you no temperature rescaling for nyx ind_rescaling=0 and 1. Why are you only upgrading those redshifts? Are you really giving more weight to these given that you set the weight to 1?

Reply:
Here I'm doing nothing. It's just the line that we could play with to scale differently the snapshots. Note that the value is set to 1

Reply:
It's val_scaling because I was playing with different options. I can move it back to ind_rescaling